CheckerPrinter/src/checkerprinter/InMemory.java

```diff
@@ -153,4 +179,20 @@ protected boolean initCheckerArgs(JsonObject optionsObject) {
             this.checkerPrinter.printDiagnosticReport(diagnosticList);
         }
     } 
+
+    //create a directory, if exists already, delete and recreate a directory
+    protected static boolean setupDir(File dir) {
```

Comment:
Maybe we don't need this method. As we can just assume there will be a `CFG` directory at the moment.